### DISCUSSION TITLE
NASA 洞察號火星探測車運作四年，電力耗盡關機除役

### DISCUSSION THREAD

**Discussion by Tuoresa** [1101-GS4719-C/Python-Community-GS4719-C#1075] [General]
於2018年5月5日發射，同年11月成功降落於火星並開始執行任務的洞察號探測器
主要任務內容為藉由地震波的觀測資料，使科學家能夠建構火星內部的成份及組成。

2022年12月21日 NASA嘗試與電量極低的洞察號聯繫未果，於是宣布洞察號已電力耗盡並關機。

每次看見這種衛星或是星體探測儀與我們失去聯繫都有一種生命殞落的悲傷
(雖然它們說白了不過是銅鐵與程式編碼的產物哈哈)

就像是卡西尼號(探測土星的衛星)的最後任務階段中，
為避免隨意墜落會傷害到其他可能存在生命的星球，
它竭盡可能進行了22次土星環軌道探索，獲得前所未有的觀測數據，
最終被高溫燃燒殆盡成為土星的一部分(好好笑 就莫名的有一種英雄壯烈感)

相關報導-1：https://www.techbang.com/posts/102602-the-battery-is-running-out-and-this-may-be-the-last-image
相關報導-2：https://www.natgeomedia.com/science/article/content-15871.html

**Comment by harrylee1971** [comment]
感覺這個探測器是一種酷東西，能在非常遠的地方還能找到地球去發出訊息，有種在茫茫人海中找到家的感覺
